src/core/utils.js

```diff
@@ -22,6 +22,8 @@ import { memoizedSampleFromSchema, memoizedCreateXMLExample } from "core/plugins
 import win from "./window"
 import cssEscape from "css.escape"
 import getParameterSchema from "../helpers/get-parameter-schema"
+import crypto from "crypto"
+import { hextob64u } from "jsrsasign"
```

Comment:
**Change request**: `jsrsasign` is [a ~250KB library](https://bundlephobia.com/result?p=jsrsasign@8.0.12), which is quite big. Can we do something lighter here?

Reply:
If I knew it was that big, I would not have included it to the dependencies. I added a method to convert base64 string to base64url-encoded, which enabled me to get rid of jsrsasign completely.